### PR TITLE
Spanish Translate  completed

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by jklingen** [comment]
Hey, awesome - thanks for creating the translation. I think I'll be able to take care of it over the weekend.

Also thanks for your donation, actually the first one I received in the Sailfish ecosystem :tada: :smiley: :+1: 

Since my Jolla 1 started to get a little unreliable after 4(!) years of everyday use and getting SFOS onto Fairphone did not work very well, I am back to Android at the moment, and not actively working on my SFOS apps, unfortunately. But of course I'll release your translation shortly :)

Thanks again! :smiley:

**Comment by ghost** [comment]
The **THANKS** are for you for creating a nice and useful app. : )
Why not a Sony Xperia X with SFOS?

**Comment by ghost** [comment]
Bump....

**Comment by jklingen** [comment]
Please bear with me, this is not forgotten - just didn't find the time last weekend. Hoping for better luck this one.

I did not buy an Xperia X because I actually wanted a fairphone :smiley: I love its idea and hardware, just hoped for better SFOS support. But I won't buy a second phone because of that ;-)

**Comment by ghost** [comment]
It is OK. Yes fairphone is nice concept and phone.

I already know that you are trying to fix camera module on  your fairphone : D

I have installed Space-Inspector v 0.6 , but not have colors on box? where is this features?

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

File: i18n/space-inspector_es.ts [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+        <translation>Saludos!</translation>
+    </message>
+    <message>
+        <source>No matter how much storage you have got - it will be full.</source>
+        <translation>No importa cuánto espacio de almacenamiento tenga, esto puede estar lleno.</translation>
+    </message>
+    <message>
+        <source>[Translator credit]</source>
+        <extracomment>If wanted, add translator info in your language, e.g. &quot;English translation by &lt;a href=\&quot;https://github.com/jklingen/\&quot;&gt;Jens Klingen&lt;/a&gt;&quot;</extracomment>
+        <translation>Si lo desea, agregue información del traductor de su lenguaje, e.j. &quot;Traducción al inglés por &lt;a href=\&quot;https://github.com/jklingen/\&quot;&gt;Jens Klingen&lt;/a&gt;&quot;</translation>
```
Comment (jklingen) [inline i18n/space-inspector_es.ts:116]: It looks like you translated the comment in the line above. This string is actually intended to (optionally) give some credit to the translator (i.e. you). So if you want, we could put there something like

    Traducción al espanol por &lt;a href=\&quot;https://github.com/Caballlero/\&quot;&gt;Carlos Gonzalez&lt;/a&gt;

or whatever else you want, and only if you want. We can also leave this empty if you're humble :smiley:

**Comment by jklingen** [comment]
Hey there, please have a look at the comment above. You can simply let me know what you would like to read in there, if it's easier than submitting another pull request.

I actually did fix the camera module, and it was pretty easy :)

The colored boxes feature is always-on, just directories all have the same color - so maybe you did not dive into a directory containing files? When looking at files, the should have different colors depending on the file extension.

File: i18n/space-inspector_es.ts (hunk 1 above)
Comment (ghost) [reply i18n/space-inspector_es.ts:116]: Sorry for that. I am trying to fix it

**Comment by ghost** [comment]
HI jklingen 
Yes I already see the colors boxes, nice thanks
I have  a issue in Ln75 this not ship translate.

**Pull request merged**